projects/plugins/jetpack/modules/masterbar.php

```diff
@@ -40,3 +40,7 @@
 if ( apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
 	require_once __DIR__ . '/masterbar/admin-menu/load.php';
 }
+
+if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
```

Comment:
There is a `jetpack_is_atomic_site` function that is used in several places, can we use it here as well? I have not seen any usage of the `IS_ATOMIC` constant, so I guess it's discouraged.

Reply:
There has been a discussion around which is better here https://github.com/Automattic/jetpack/pull/19762#discussion_r631209263

Reply:
Yeah, there was also a discussion on Slack a few weeks ago about this so I thought that IS_ATOMIC would be faster. Anyway, let's use `jetpack_is_atomic_site`.